config/fxdata/magic.cfg

```diff
@@ -1099,6 +1104,8 @@ ShotSoundPriority = 3
 HitCreatureSound = 137 1
 WithstandHitAgainst = WALL DOOR
 Properties = REBOUND_IMMUNE WIND_IMMUNE NO_STUN
+ShotType = 2
```

Comment:
I assume ShotType = 2 is the defaut for all shots?

Reply:
no 0, but I'll finish it up tomorrow, including some comments